guessit/rules/common/pattern.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Pattern utility functions
+"""
+
+
+def is_enabled(context, name):
```

Comment:
It would be better to implement is_disabled to make this consistent with rebulk API and avoid using not operator when calling this.